interface/patient_file/transaction/transactions.php

```diff
@@ -48,28 +44,22 @@ function toggle( target, div ) {
 </head>
 
 <body class="body_top">
-    <table>
-    <tr>
-        <td>
-            <span class="title"><?php echo htmlspecialchars( xl('Patient Transactions'), ENT_NOQUOTES); ?></span>&nbsp;</td>
-        <td>
-            <!-- Define CSS Buttons -->
-            <a href="add_transaction.php" class="css_button" onclick="top.restoreSession()">
-            <span><?php echo htmlspecialchars( xl('Add'), ENT_NOQUOTES); ?></span></a>
-        </td>
-        <td>
-            <a href="print_referral.php" onclick="top.restoreSession()" class="css_button" >
-            <span><?php echo htmlspecialchars( xl('View Blank Referral Form'), ENT_NOQUOTES); ?></span></a>
-        </td>
-    </tr>
-    </table>
+    <div class="page-header">
+        <h1><?php echo xl('Patient Transactions');?></h1>
+    </div>
+    <div class="btn-group">
+        <a href="add_transaction.php" class="btn btn-default btn-add" onclick="top.restoreSession()">
+            <?php echo htmlspecialchars( xl('Add'), ENT_NOQUOTES); ?></a>
+        <a href="print_referral.php" onclick="top.restoreSession()" class="btn btn-default" >
```

Comment:
could use the btn-transmit icon here (that icon seems to look well when there's not another more specific one)